docs/release.md

```diff
@@ -115,137 +114,130 @@ Similarly, you can use the SHA256 digest of the provenance to download the
-    },
-    "publicKey": "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUR3VENDQTBlZ0F3SUJBZ0lVY2NGTFRRVnZPREsvamRoVlMrT3N4NUFDWXVVd0NnWUlLb1pJemowRUF3TXcKTnpFVk1CTUdBMVVFQ2hNTWMybG5jM1J2Y21VdVpHVjJNUjR3SEFZRFZRUURFeFZ6YVdkemRHOXlaUzFwYm5SbApjbTFsWkdsaGRHVXdIaGNOTWpJeE1URTRNVGN4TmpJeFdoY05Nakl4TVRFNE1UY3lOakl4V2pBQU1Ga3dFd1lICktvWkl6ajBDQVFZSUtvWkl6ajBEQVFjRFFnQUVETTRDRFpJMWFYRkZqQmc1d0xaSVR5VjB0RklFL3VRNlo5cXIKNFkwRlBoaWt0U3hQbzRwWDJ4L2NVVjgyMjR6WjlaMTZaUUtSQ0RPc0tIdXA5bmpScUtPQ0FtWXdnZ0ppTUE0RwpBMVVkRHdFQi93UUVBd0lIZ0RBVEJnTlZIU1VFRERBS0JnZ3JCZ0VGQlFjREF6QWRCZ05WSFE0RUZnUVVXdWkrCjJqTjBnL21aY1BUazhpOWwrZEY5eXpzd0h3WURWUjBqQkJnd0ZvQVUzOVBwejFZa0VaYjVxTmpwS0ZXaXhpNFkKWkQ4d2dZUUdBMVVkRVFFQi93UjZNSGlHZG1oMGRIQnpPaTh2WjJsMGFIVmlMbU52YlM5emJITmhMV1p5WVcxbApkMjl5YXk5emJITmhMV2RwZEdoMVlpMW5aVzVsY21GMGIzSXZMbWRwZEdoMVlpOTNiM0pyWm14dmQzTXZaMlZ1ClpYSmhkRzl5WDJkbGJtVnlhV05mYzJ4ellUTXVlVzFzUUhKbFpuTXZkR0ZuY3k5Mk1TNHlMakV3T1FZS0t3WUIKQkFHRHZ6QUJBUVFyYUhSMGNITTZMeTkwYjJ0bGJpNWhZM1JwYjI1ekxtZHBkR2gxWW5WelpYSmpiMjUwWlc1MApMbU52YlRBU0Jnb3JCZ0VFQVlPL01BRUNCQVJ3ZFhOb01EWUdDaXNHQVFRQmc3OHdBUU1FS0RNNU9ERXdORGRrCllqaG1PV1kyTVRNMU5XRmlPVE13TlRSbVlUQXpPV001TURjd09UUmpZelV3SXdZS0t3WUJCQUdEdnpBQkJBUVYKUW5WcGJHUWdRV3hzSUZCeWIzWmxibUZ1WTJWek1CMEdDaXNHQVFRQmc3OHdBUVVFRDNCeWIycGxZM1F0YjJGcgpMMjloYXpBZEJnb3JCZ0VFQVlPL01BRUdCQTl5WldaekwyaGxZV1J6TDIxaGFXNHdnWWtHQ2lzR0FRUUIxbmtDCkJBSUVld1I1QUhjQWRRRGRQVEJxeHNjUk1tTVpIaHlaWnpjQ29rcGV1TjQ4cmYrSGluS0FMeW51amdBQUFZU0wKdmNuZkFBQUVBd0JHTUVRQ0lCdk1lOXlMeVJkVk43dm03TDRyUjlDZjgzVGpoc29GU25KbC9JcTlnVktsQWlCKwpxWXUrQzA1bEx0NDI4YWN3RFFDcXlsZ0MvVlJPMWtDaTZhbUtwVGlXM0RBS0JnZ3Foa2pPUFFRREF3Tm9BREJsCkFqRUFtKzdsL28zSDBZV0VLLzNXeDNkV1ZWUWFNR1NBUXQvbitxUlJnRWRia25yaXUwNlNtQWNPcitwamV6R2gKMmxidEFqQVF4bnVuNTd6djNLNkUxREkydEg1cUk1UzBZQk40MmdraVBpOWlvRXo4NU8rcnZlNXpGOS9ObXNjaApUS1RheEN3PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg=="
+    }
   }
 }
 ```
 
-In this case, the provenance is in the `Attestation` field. The response in this
-case does not contain the verification, because `rekor-cli` already verifies the
-LogEntry and its inclusion in Rekor. However, it contains other details,
-including the `Body.IntotoObj` object. The `payloadHash` in this object is the
-SHA256 hash of the provenance.
+Note that the provenance is included in the `Attestation` field. Compared to the
+Sigstore bundle above, the response in this case does not contain the
+`verificationMaterial`, because `rekor-cli` already verifies the LogEntry and
+its inclusion in Rekor.
+
+## What is next?
+
+Once the team is ready to transparently release a binary, they have to endorse
+it, using the
+[`endorser tool`](https://github.com/project-oak/transparent-release/tree/main/internal/endorser).
+This tool takes as input the hash of a binary and one or more provenances, then
+it verifies the provenances, and generates an endorsement statement. For more
+details on using the endorser tool see
+[these instructions (currently WIP)](https://github.com/project-oak/transparent-release/tree/main/cmd).
```

Comment:
I think the fact that they are WIP should be on the instructions themselves, not where they are linked from

Reply:
Removed `(currently WIP)`.